model/famis_models.ts

```diff
@@ -19,6 +19,24 @@ export interface SpaceClass {
     UpdatedByName?: string;
 }
 
+export interface Department {
+    Id: number,
```

Comment:
This should be an optional since you may not always Select `Id`

Reply:
done